srtcore/epoll.cpp

```diff
@@ -475,7 +549,9 @@ int CEPoll::wait(const int eid, set<SRTSOCKET>* readfds, set<SRTSOCKET>* writefd
             if (!ed.flags(SRT_EPOLL_ENABLE_EMPTY) && ed.watch_empty() && ed.m_sLocals.empty())
             {
                 // Empty EID is not allowed, report error.
-                throw CUDTException(MJ_NOTSUP, MN_INVAL);
+                //throw CUDTException(MJ_NOTSUP, MN_INVAL);
```

Comment:
Decided to introduce a new type of error and change it in this place because it doesn't match the general interpretation of the INVAL error. INVAL usually means that you have used wrong ID. This problem means that the call to `wait`-function would block forever as all sockets were unsubscribed. This isn't something that could have been prevented by the user, or at least this kind of error is wanted, as flags didn't turn it off.